### DISCUSSION TITLE
Can't log in to Budibase

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#13378] [Help]
hello,
  I can't log in to Budibase.
Oh, other users can't access the app.
But users who have logged in can use normally.
I don't know what happened. Yesterday it still worked normally.

I tried adding a new user, but the new user logs into the App.

![image](https://github.com/Budibase/budibase/assets/93240913/441d4686-2dcc-4f6c-8bab-030725da5c65)

**Comment by ConorWebb96** [comment]
Hey @LTCTHITA,

Looking at your screenshot you do indeed seem to be logged in? Simply click on the Budibase icon and it should redirect you to your builder.

Let me know if this helps.
